Makefile.toml

```diff
@@ -533,6 +534,71 @@ cargo run -p risingwave_cmd_all \
           -- playground
 '''
 
+[tasks.all-in-one]
+category = "RiseDev - Start/Stop"
+description = "🌟 Start a RisingWave all-in-one instance"
+dependencies = ["build"]
+script = '''
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+RUST_BACKTRACE=1 \
+cargo run -p risingwave_cmd_all \
+          --profile "${RISINGWAVE_BUILD_PROFILE}" \
+          ${RISINGWAVE_FEATURE_FLAGS} \
+          -- monolithic-mode $@
+'''
+
+# TODO(kwannoel): Support `tasks.all-in-one-dev` as well.
+# Then we can reuse risedev to start all-in-one peripheral services, e.g. compactor, etcd, kafka, etc...
+[tasks.all-in-one-demo]
+category = "RiseDev - Start/Stop"
+description = "🌟 Start a RisingWave all-in-one instance in development mode, with demo parameters"
+dependencies = ["build"]
+script = '''
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+RUST_BACKTRACE=1 \
+cargo run -p risingwave_cmd_all \
+          --profile "${RISINGWAVE_BUILD_PROFILE}" \
+          ${RISINGWAVE_FEATURE_FLAGS} \
+          -- monolithic-mode \
+               --meta-opts=" \
+                  --listen-addr 127.0.0.1:5690 \
+                  --advertise-addr 127.0.0.1:5690 \
+                  --dashboard-host 127.0.0.1:5691 \
+                  --prometheus-host 127.0.0.1:1250 \
+                  --connector-rpc-endpoint 127.0.0.1:50051 \
+                  --backend mem \
+                  --state-store hummock+memory \
+                  --data-directory hummock_001 \
+                  --config-path src/config/all-in-one-example.toml \
+                  --dashboard-ui-path /Users/noelkwan/projects/risingwave/.risingwave/ui" \
```

Comment:
Seems not a good idea to hardcode the path. 😕

Reply:
Oops I copied it from risedev.log.

I will just remove it, it's not necessary i think.